docs/quickstart.rst

```diff
@@ -87,7 +116,12 @@ Once you have a recent Python 3 version of ``pip``, you should be able to upgrad
 
 .. code-block:: bash
 
-    pip install --upgrade setuptools
+    $ pip install --upgrade setuptools
+
+    OR
+
+    $ sudo pip3 install --upgrade setuptools
```

Comment:
Because it didn't work for me without the `sudo` and I was not using the `virtualenv` or `conda`, and running this on base VM. I can change that to `pip3 install --upgrade setuptools` as well to keep it consistent, and if the user encounters a permissions issue they can run it with `sudo` which is pretty straight forward. Same for the next subsection, because with virtualenv, I don't need `sudo` permissions.